keyboards/boardsource/technik_o/info.json

```diff
@@ -1,62 +1,144 @@
 {
-    "keyboard_name": "Technik-O",
-    "url": "https://boardsource.xyz",
-    "maintainer": "Boardsource",
-    "layouts": {
-        "LAYOUT_ortho_4x12": {
-            "layout": [
-                { "label": "K01", "x": 0, "y": 0 },
-                { "label": "K02", "x": 1, "y": 0 },
-                { "label": "K03", "x": 2, "y": 0 },
-                { "label": "K04", "x": 3, "y": 0 },
-                { "label": "K05", "x": 4, "y": 0 },
-                { "label": "K06", "x": 5, "y": 0 },
-                { "label": "K07", "x": 6, "y": 0 },
-                { "label": "K08", "x": 7, "y": 0 },
-                { "label": "K09", "x": 8, "y": 0 },
-                { "label": "K010", "x": 9, "y": 0 },
-                { "label": "K011", "x": 10, "y": 0 },
-                { "label": "K012", "x": 11, "y": 0 },
-
-                { "label": "K11", "x": 0, "y": 1 },
-                { "label": "K12", "x": 1, "y": 1 },
-                { "label": "K13", "x": 2, "y": 1 },
-                { "label": "K14", "x": 3, "y": 1 },
-                { "label": "K15", "x": 4, "y": 1 },
-                { "label": "K16", "x": 5, "y": 1 },
-                { "label": "K17", "x": 6, "y": 1 },
-                { "label": "K18", "x": 7, "y": 1 },
-                { "label": "K19", "x": 8, "y": 1 },
-                { "label": "K110", "x": 9, "y": 1 },
-                { "label": "K111", "x": 10, "y": 1 },
-                { "label": "K112", "x": 11, "y": 1 },
-
-                { "label": "K21", "x": 0, "y": 2 },
-                { "label": "K22", "x": 1, "y": 2 },
-                { "label": "K23", "x": 2, "y": 2 },
-                { "label": "K24", "x": 3, "y": 2 },
-                { "label": "K25", "x": 4, "y": 2 },
-                { "label": "K26", "x": 5, "y": 2 },
-                { "label": "K27", "x": 6, "y": 2 },
-                { "label": "K28", "x": 7, "y": 2 },
-                { "label": "K29", "x": 8, "y": 2 },
-                { "label": "K210", "x": 9, "y": 2 },
-                { "label": "K211", "x": 10, "y": 2 },
-                { "label": "K212", "x": 11, "y": 2 },
-
-                { "label": "K31", "x": 0, "y": 3 },
-                { "label": "K32", "x": 1, "y": 3 },
-                { "label": "K33", "x": 2, "y": 3 },
-                { "label": "K34", "x": 3, "y": 3 },
-                { "label": "K35", "x": 4, "y": 3 },
-                { "label": "K36", "x": 5, "y": 3 },
-                { "label": "K37", "x": 6, "y": 3 },
-                { "label": "K38", "x": 7, "y": 3 },
-                { "label": "K39", "x": 8, "y": 3 },
-                { "label": "K310", "x": 9, "y": 3 },
-                { "label": "K311", "x": 10, "y": 3 },
-                { "label": "K312", "x": 11, "y": 3 }
-            ]
-        }
+  "manufacturer": "Boardsource",
+  "keyboard_name": "Technik-O",
+  "maintainer": "waffle87",
+  "bootloader": "atmel-dfu",
```

Comment:
dev board change here too?

Reply:
i don't think so — this one uses an integrated microcontroller.

Reply:
Sorry, you're right!